src/utils/useUpdateWithSetter.ts

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { CollectionOptions } from '../Components';
+import {CollectionOptions} from '../Components';
```

Comment:
these changes are because I ran `yarn run prettier`; the build errors are expected and should be resolved once the above TODOs are completed